src/nimUtils.ts

```diff
@@ -75,18 +75,17 @@ export function getBinPath(tool: string): string {
         _pathesCache[tool] = pathparts.map(dir => path.join(dir, correctBinname(tool))).filter(candidate => fs.existsSync(candidate))[0];
         if (process.platform !== 'win32') {
             try {
-                let buff;
+                let nimPath;
                 if (process.platform === 'darwin') {
-                    // dirty hack because the version of readlink that ships with osx doesn't support the -f flag
-                    // from https://stackoverflow.com/questions/1055671/how-can-i-get-the-behavior-of-gnus-readlink-f-on-a-mac#4031502
-                    buff = cp.execFileSync('python', ['-c', 'import os,sys;print(os.path.realpath(sys.argv[1]))', _pathesCache[tool]]);
+                    nimPath = _pathesCache[tool].slice(0, _pathesCache[tool].length - 3) + cp.execFileSync('readlink', [_pathesCache[tool]]).toString().trim();
```

Comment:
As far as I can tell, ```readlink``` always returns a path relative to the parent of the symlink and thus will never have a ```/``` in front. 
But as a general question, why do you need to resolve the symlink for ```nim``` anyway? It works totally fine for me if I use the path to nim provided by ```PATH```.